bitnami/rabbitmq/README.md

```diff
@@ -284,192 +284,82 @@ $ helm install my-release -f values.yaml bitnami/rabbitmq
-          "tags": "administrator"
-        }
-      ],
-      "vhosts": [
-        {
-          "name": "/"
-        }
-      ]
-    }
-```
+Because definitions may contain RabbitMQ credentials, [store the JSON as a Kubernetes secret](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-files-from-a-pod). Within the secret's data, choose a key name that corresponds with the desired load definitions filename (i.e. `load_definition.json`) and use the JSON object as the value.
 
-Then, specify the `load_definitions` property as an `extraConfiguration` pointing to the load definition file path within the container (i.e. `/app/load_definition.json`) and set `loadDefinition.enable` to `true`. Any load definitions specified will be available within in the container at `/app`.
+Next, specify the `load_definitions` property as an `extraConfiguration` pointing to the load definition file path within the container (i.e. `/app/load_definition.json`) and set `loadDefinition.enable` to `true`. Any load definitions specified will be available within in the container at `/app`.
 
-> Loading a definition will take precedence over any configuration done through [Helm values](#parameters).
+> NOTE: Loading a definition will take precedence over any configuration done through [Helm values](#parameters).
 
-If needed, you can use `extraSecrets` to let the chart create the secret for you. This way, you don't need to manually create it before deploying a release. These secrets can also be templated to use supplied chart values. For example:
+If needed, you can use `extraSecrets` to let the chart create the secret for you. This way, you don't need to manually create it before deploying a release. These secrets can also be templated to use supplied chart values.
 
-```yaml
-auth:
-  password: CHANGEME
-extraSecrets:
-  load-definition:
-    load_definition.json: |
-      {
-        "users": [
-          {
-            "name": "{{ .Values.auth.username }}",
-            "password": "{{ .Values.auth.password }}",
-            "tags": "administrator"
-          }
-        ],
-        "vhosts": [
-          {
-            "name": "/"
-          }
-        ]
-      }
-loadDefinition:
-  enabled: true
-  existingSecret: load-definition
-extraConfiguration: |
-  load_definitions = /app/load_definition.json
-```
+Refer to the chart documentation for [more information and configuration examples of loading custom definitions](https://docs.bitnami.com/kubernetes/infrastructure/rabbitmq/configuration/load-files/).
 
-### LDAP
+### Configure LDAP support
 
-LDAP support can be enabled in the chart by specifying the `ldap.` parameters while creating a release. The following parameters should be configured to properly enable the LDAP support in the chart.
+LDAP support can be enabled in the chart by specifying the `ldap.*` parameters while creating a release. The following parameters should be configured to properly enable the LDAP support in the chart.
```

Comment:
This would be repeated in the README and docs. Can we have here the parameters and in the Docs a more detailed example?

Reply:
Done